src/Pos/Ssc/Type.hs

```diff
@@ -0,0 +1,14 @@
+-- | Global export of the @SSC@ type. Exported in order to be able to switch to 
+-- different type-level implementations.
+
+module Pos.Ssc.Type
+       ( SscType
+       ) where
+
+import Pos.Ssc.GodTossing (SscGodTossing)
+--import Pos.Ssc.NistBeacon (SscNistBeacon)
+
+-- | We can also use CPP flags that I proposed that would enable switching between 
+-- implementations without changing the code.
+type SscType = SscGodTossing
```

Comment:
This is used (and should be used) only by wallet. So this module should be somewhere in `Pos.Wallet`. We want to keep node itself (without wallet) without knowledge of particular `ssc`.

Also I would rename it to `WalletSsc[Type]`.